src/firebase-import.js

```diff
@@ -80,29 +82,74 @@ function promptToContinue(ref, next) {
   }
 }
 
+function getJsonFromFile(file, callback) {
+  var firstChar;
+  var rs = fs.createReadStream(file);
+  rs.on('data', function(chunk) {
+    var s = chunk.toString().trim();
+    if (s !== "") {
+      rs.close();
+    }
+    firstChar = s[0];
+  })
+  .on('close', function() {
+    if (firstChar === "[" || firstChar === "{") {
+      var jsonStream;
+      var onFunc;
+      var json;
+      if (firstChar === "[") {
+        console.log("Reading json as an array...");
```

Comment:
I'd remove these log statements, since I don't think they're relevant to the user. :-)